web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -198,68 +198,77 @@ private List<HttpSessionListener> getHttpSessionListeners() {
 	@Override
 	public void contextInitialized(ServletContextEvent event) {
 		log.debug("Starting the OpenMRS webapp");
-		
+	
 		try {
-			// validate the current JVM version
 			OpenmrsUtil.validateJavaVersion();
-			
+	
 			ServletContext servletContext = event.getServletContext();
-			
-			// pulled from web.xml.
+	
 			loadConstants(servletContext);
-			
-			// erase things in the dwr file
```

Comment:
we should keep these comments in this method, only apply your changes without deleting them.

Reply:
@JayadityaGit did you see this?